src/Form/ConfigPrForm.php

```diff
@@ -146,27 +148,40 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     }
     $user = User::load($this->currentUser()->id());
     $authToken = $user->field_config_pr_auth_token->value;
+    $appPassword = 'password';
     if (empty($authToken)) {
       $uid = \Drupal::currentUser()->id();
       \Drupal::messenger()->addError($this->t('Config Pull Request Auth Token missing!'));
       $response = new RedirectResponse('/user/' . $uid . '/edit');
       $response->send();
     }
     else {
-      $this->repoController->setAuthToken($authToken);
+      // @todo: define $appPassword.
```

Comment:
All of these changes to add user/pass need to be in the hook_form_alter of the submodule